src/components/role-info-panel/role-info-panel.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { useEffect } from 'react';
+import { Grid, Typography } from '@material-ui/core';
+import CheckIcon from '@material-ui/icons/Check';
+import { useDispatch, useSelector } from 'react-redux';
+import { ApiRole, ApiRosterColumnInfo } from '../../models/api-response';
+import useStyles, { ScrollHeightParam } from './role-info-panel.styles';
+import { parsePermissions } from '../../utility/permission-set';
+import { Notification } from '../../actions/notification.actions';
+import { Roster } from '../../actions/roster.actions';
+import { RosterSelector } from '../../selectors/roster.selector';
+import { NotificationSelector } from '../../selectors/notification.selector';
+import { UserSelector } from '../../selectors/user.selector';
+import { RoleDataTable } from './role-table';
+import NegativeMarginScrollFix from './negative-margin-scroll-fix';
+
+const Header = ({ text }: { text: string }) => <Typography className={useStyles().roleHeader}>{text}:</Typography>;
```

Comment:
This was the start of a potentially larger refactor. If you search the code for `lineHeight: '24px'` you'll find twelve places that are almost exactly copy and pasted. The idea was to have a reusable component to use everywhere instead of redeclaring the CSS repeatedly.